doc/releases/pacific.rst

```diff
@@ -2,6 +2,41 @@
 Pacific
 =======
 
+v16.2.10 Pacific
+================
+
+This is a hotfix release that resolves two security flaws.
+
+Notable Changes
+---------------
+* Users who were running OpenStack Manila to export native CephFS, who
```

Comment:
@zdover23 I would remove the comma(s) and add an _and_ before _who upgraded_.

Reply:
Both of these changes have been made.